capstone/src/main/java/com/google/step/servlets/RestaurantServlet.java

```diff
@@ -71,7 +77,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     // Only available if the user is logged in.
     if (!userService.isUserLoggedIn()) {
-      response.setStatus(HttpServletResponse.SC_FORBIDDEN);
+      response.sendError(HttpServletResponse.SC_FORBIDDEN);
```

Comment:
Wouldn't this just put that error in the entire page and not allow us to make it pretty with the UI?

Reply:
I don't believe so, but I could be wrong. I think `sendError()` simply sends an HTTP response body that sends back the error code and a default shortened message the front end still decides what to do with it. If I'm not mistaken, `setStatus()` doesn't actually change them message. I'll look into it, though.

Reply:
So I did a bit of digging and it seems that you're right. I tested both out, and they seemed to do the exact same thing on the local server. When I did a bit of digging, I found out that `sendError()` only redirects if the servlet container specifies what should happen in the case of a specific kind of error. In this case, we didn't specify a redirect url anywhere, so nothing happened. In the deployed app, App Engine probably has it's own rules regarding the matter. I'll change it back because the original functionality was actually what we wanted.